.github/workflows/shared.yml

```diff
@@ -148,6 +148,7 @@ jobs:
       - name: Upload binary to bootstrap.urbit.org
         if: ${{ inputs.upload }}
         run: |
+          echo "${{ inputs.pace }}" > ./PACE
           version="$(cat ./VERSION)"
           bazel build :version
           sha_version=$(grep 'URBIT_VERSION *".*"' ./bazel-bin/version.h \
```

Comment:
It doesn't fit in this PR, but determining the version can actually simplify to:
```bash
bazel build :version_str
sha_version=$(cat bazel-bin/version)
```
The `grep` pipeline was necessary before we had `//:version_str`.